scripts/libraries/generated-dependencies.ts

```diff
@@ -0,0 +1,26 @@
+import { EncodableArray, encodeMethod } from '@zoltu/ethereum-abi-encoder';
+import { JsonRpc } from '@zoltu/ethereum-types';
+import { keccak256 } from '@zoltu/ethereum-crypto';
+
+interface Log {
+	readonly topics: ReadonlyArray<bigint>
+	readonly data: Uint8Array
+}
+interface TransactionReceipt {
+	readonly status: boolean
+	readonly logs: Iterable<Log>
+}
+
+export class DependenciesImpl {
```

Comment:
What's the diff between this and TestDependencies?

Reply:
`TestDependencies` at the moment includes some code for special casing gas estimating.  Eventually it may also include code for manual mining instead of automatic mining if we find that is necessary for DSR testing.